tests/integrated/test-multigrid_laplace_sheared-grid/test_multigrid_laplace.cxx

```diff
@@ -0,0 +1,325 @@
+/**************************************************************************
```

Comment:
This model is identical to the file in tests/integrated/test-multigrid_laplace/test_multigrid_laplace.cxx

Can the two test directories be merged just with different data directories?